firebase-storage/src/main/java/com/google/firebase/storage/network/connection/package-info.java

```diff
@@ -13,4 +13,7 @@
 // limitations under the License.
 
 /** @hide */
-package com.google.firebase.storage.network.connection;
\ No newline at end of file
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
I take it the one on network/ doesn't cascade to network/connection?

Reply:
No, unfortunately it doesn't.